source/developers-guide/plugin-quick-start/index.md

```diff
@@ -525,3 +595,520 @@ Possible settings for the parameter `caches` contains each context as constant.
 `...Context::CACHE_TAG_ROUTER`
 `...Context::CACHE_TAG_TEMPLATE`
 `...Context::CACHE_TAG_THEME`
+
+## Container Configuration
+
+The [Symfony DependencyInjection Component](http://symfony.com/doc/current/components/dependency_injection/introduction.html)
```

Comment:
please link to the symfony 2.8 doc, as this is the version we use

Reply:
d